src/test/java/org/kiwiproject/champagne/dao/TestDbObjects.java

```diff
@@ -0,0 +1,144 @@
+package org.kiwiproject.champagne.dao;
+
+import org.jdbi.v3.core.Handle;
+import org.kiwiproject.champagne.model.AuditRecord;
+import org.kiwiproject.champagne.model.DeploymentEnvironment;
+import org.kiwiproject.champagne.model.User;
+import org.kiwiproject.champagne.model.AuditRecord.Action;
+import org.kiwiproject.champagne.model.manualdeployment.DeploymentTaskStatus;
+import org.kiwiproject.champagne.model.manualdeployment.Release;
+import org.kiwiproject.champagne.model.manualdeployment.ReleaseStage;
+import org.kiwiproject.champagne.model.manualdeployment.ReleaseStatus;
+import org.kiwiproject.champagne.model.manualdeployment.Task;
+import org.kiwiproject.champagne.model.manualdeployment.TaskStatus;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class TestDbObjects {
```

Comment:
So, when you asked me what we "normally" call this class, I said `TestObjects` (no `Db` in the name). The reason is because, in general, there are methods to create the POJOs, and methods to perform DB crud. So, it might contain a `newAuditRecord()` _and_ a `insertAuditRecord(Handle)`.

We tend have the following convention:
* `newXxx` means create a new _instance_ of `Xxx`. There can be overloads that accept custom parameters
* `insertXxx(Handle, Xxx)` means (assuming JDBI) to do a SQL _insert_ of the given `Xxx` instance
* `updateXxx(Handle, Xxx) means to do a SQL _update_ of the given `Xxx` instance

I stay away from the word "save" unless I really mean it will perform an insert _or_ and update.

I think it's redundant to include the word "Test" in all the methods, since the class is named `TestObjects`.

Reply:
If you don't actually need the separate `newXxx` methods right now, that's fine. But still recommend renaming to just `TestObjects`, removing `Test` from the method names, and changing them to `insertXxx`, etc.